Source/Core/EarthOrientationParameters.js

```diff
@@ -95,7 +97,7 @@ define([
         } else if (defined(options.url)) {
             // Download EOP data.
             var that = this;
-            this._downloadPromise = when(loadJson(options.url), function(eopData) {
+            this._downloadPromise = when(loadJson(new Resource({url: options.url})), function(eopData) {
```

Comment:
This file can be reverted, right?  No reason to manually construct the Resource since `loadJson` still takes a string which makes for cleaner code.